jwst/ramp_fitting/ramp_fit.py

```diff
@@ -234,124 +236,108 @@ def ols_ramp_fit(model, buffsize, save_opt, readnoise_model, gain_model,
+    c_rates = slopes/effintim
 
     # Values in err_cube have been 1./variance, so take reciprocal
     #    of non-zero values to write
-    err_cube[err_cube <= 0.] = MIN_ERR  # for pixels having no signal
-    err_cube = 1. / err_cube # has no zero values
+    err_cube [ err_cube <= 0. ] = MIN_ERR  # for pixels having no signal
+    err_cube = 1./err_cube # has no zero values
 
     # Compress all integration's dq arrays to create 2D PIXELDDQ array for
     #   primary output
-    final_pixeldq = dq_compress_final(dq_int, n_int)
-
-    # If either skip_i or skip_f are > 0, include the FRAMES_SKIPPED flag
-    #   within the output 2D PIXELDQ and, if there is more than one
-    #   integration, also include this flag within the integration-specific dq
-    #   output cube
-    if (skip_i + skip_f > 0):
-        final_pixeldq = np.bitwise_or(dqflags.pixel['FRAMES_SKIPPED'],
-                                       final_pixeldq)
-        if n_int > 1:
-            dq_int = np.bitwise_or(dqflags.pixel['FRAMES_SKIPPED'], dq_int)
+    final_pixeldq = dq_compress_final( dq_int, n_int )
 
     if n_int > 1:
-        int_model = utils.output_integ(model, slope_int, err_int, dq_int,
-                                        effintim)
+        int_model = utils.output_integ( model, slope_int, err_int, dq_int,
+                                        effintim )
     else:
         int_model = None
 
     tstop = time.time()
 
-    log_stats(c_rates)
+    log_stats( c_rates )
 
-    log.debug('Instrument: %s' % (instrume))
-    log.debug('Number of pixels in 2D array: %d' % (npix))
-    log.debug('Shape of 2D image: (%d, %d)' % (imshape))
-    log.debug('Shape of data cube: (%d, %d, %d)' % (cubeshape))
-    log.debug('Buffer size (bytes): %d' % (buffsize))
-    log.debug('Number of rows per buffer: %d' % (nrows))
-    log.info('Number of groups per integration: %d' % (nreads))
-    log.info('Number of integrations: %d' % (n_int))
-    log.info('Number of initial groups skipped: %d' % (skip_i))
-    log.info('Number of final groups skipped: %d' % (skip_f))
-    log.debug('The execution time in seconds: %f' % (tstop - tstart))
+    log.debug('Instrument: %s' %(instrume))
+    log.debug('Number of pixels in 2D array: %d' %(npix))
+    log.debug('Shape of 2D image: (%d, %d)' %(imshape))
+    log.debug('Shape of data cube: (%d, %d, %d)' %(orig_cubeshape))    
+    log.debug('Buffer size (bytes): %d' %(buffsize))
+    log.debug('Number of rows per buffer: %d' %(nrows))
+    log.info('Number of groups per integration: %d' %(orig_nreads))
+    log.info('Number of integrations: %d' %(n_int))
+    log.info('Number of initial groups skipped: %d' %(skip_i))
+    log.info('Number of final groups skipped: %d' %(skip_f))
+    log.debug('The execution time in seconds: %f' %(tstop - tstart))
 
     # Create new model...
-    new_model = datamodels.ImageModel(data=c_rates.astype(np.float32),
+    new_model = models.ImageModel( data=c_rates.astype(np.float32),
```

Comment:
Whoops - gotta change "models" back to "datamodels" so that it'll work in the new package structure.